packages/mds-api-helpers/tests/index.spec.ts

```diff
@@ -14,13 +14,27 @@
  * limitations under the License.
  */
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
 import { uuid } from '@mds-core/mds-utils'
-import test from 'unit.js'
-import { parseRequest } from '../index'
+import { asJsonApiLinks, parseRequest } from '../index'
 
 const arraysEqual = <T>(array1?: T[], array2?: T[]) => JSON.stringify(array1) === JSON.stringify(array2)
 
 describe('mds-api-helpers Tests', () => {
+  describe('asJsonApiLinks test', () => {
+    it('outputs links', () => {
+      const uuid1 = uuid()
```

Comment:
Made the same note on Russen's PR, but could you make this `provider_id` instead of uuid1? This may just be resolved by merging in `develop`.